front-end/src/view/components/Button.tsx

```diff
@@ -0,0 +1,13 @@
+import { ArrowRight } from 'phosphor-react'
+
+interface ButtonProps {
+    name: string
+}
+
+export default function Button({ name }: ButtonProps) {
+    return (
+        <>
```

Comment:
NIT: Since this component it's only returning one element, the fragment is not necessary.